src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/stores/AbstractFormStore.js

```diff
@@ -124,7 +124,11 @@ function collectTagPathsWithPriority(
             continue;
         }
 
-        if (types && Object.keys(types).length > 0 && data[key]) {
+        if (types
+            && Object.keys(types).length > 0
```

Comment:
maybe we should test if the `type` is `block` rather than this? looks like the handling below is for blocks, right? 🤔

Reply:
It would work for any other type as well, that saves the different type as an array. I think I would not be too restrictive here, since this is a part of the code you can't (really) touch as 3rd party developer.